index.html

```diff
@@ -7,15 +7,35 @@
   </head>
 
   <body>
-    <h1>Tic Tac Toe</h1>
+    <h1>Tic Tac Toad</h1>
+    <h2 id="winners"></h2>
     <div id="tic-tac-toe"></div>
   </body>
   <script src="https://code.jquery.com/jquery-2.2.0.min.js"></script>
   <script src="tic-tac-toe.js"></script>
   <script type="text/javascript">
     $(document).on('ready', function() {
-      console.log('create and begin the game here!');
+      var game = new TicTacToe();
+      game.play;
```

Comment:
It seems like this isn't necessary. Will this call a function or just return the function itself?
It also looks like you have `game.play()` inside of the TicTacToe "constructor" and I think this is what is really kicking off the game play.